backend/schema/endpoints/streams.json

```diff
@@ -20,9 +20,10 @@
       "minimum": 1,
       "maximum": 65535
     },
-    "forward_ip": {
+    "forwarding_host": {
```

Comment:
To enable validation of only a domain or an IP, this can be:
```
"forwarding_host": {
  "oneOf": [
    {
      "$ref": "../definitions.json#/definitions/domain_name"
    },
    {
      "type": "string",
      "format": "ipv4"
    },
    {
      "type": "string",
      "format": "ipv6"
    }
},
```